lib/widgets/matrix.dart

```diff
@@ -208,6 +208,7 @@ class MatrixState extends State<Matrix>
         if (activeClient == null) return;
         final result = await setActiveClient(activeClient);
         if (result.isSuccess) {
+          setUpToMServicesInLogin(activeClient);
```

Comment:
what is this case?

Reply:
First login is matrix.org not support ToM and add another account have supports ToM => handle config tom.

Before changing it we missing this case

Reply:
IMO, all is setup inside `_registerSubs` method? what is the problem?